package.json

```diff
@@ -8,10 +8,10 @@
   "license": "MIT",
   "scripts": {
     "build:all": "lerna run build --stream",
-    "build": "lerna run build --scope beachball",
-    "test": "lerna run test --scope beachball",
+    "build": "yarn workspace beachball build",
+    "test": "yarn workspace beachball test",
     "postbuild:all": "rimraf docs && ncp packages/website/public docs",
-    "start": "lerna run start",
+    "start": "yarn workspace beachball start",
```

Comment:
Curious what drove this just to know if I should make this change in flamegrill too.

Reply:
This is a realization that most of the time i'm only building ONE package. I also don't expect us to have more than one package in this repo... so Lerna isn't really needed honestly.